docs/encyclopedia/temporal-nexus-overview.mdx

```diff
@@ -0,0 +1,118 @@
+toc_max_heading_level: 4
+slug: /temporalnexus/overview
+keywords:
+  - modular architecture
+  - Namespace boundaries
+  - durable execution
+tags:
+  - distributed systems
+  - integration
+  - temporal
+---
+
+## Core Components {#nexus-core-components}
+
+Nexus is designed to connect durable executions across team and Namespace boundaries with an integrated Temporal SDK experience, built-in Nexus Machinery that processes Nexus RPC calls on your behalf, and promotes a more modular architecture for sharing a subset of your team’s capabilities via a well-defined Nexus Service API contract.
+
+Nexus Services are exposed from a Nexus Endpoint that is created in the Nexus API Registry, along with service documentation that enable others to use the Nexus Services provided by a Nexus Endpoint. This enables teams to compose Nexus Services into their Workflows, both within and across Namespace boundaries.
+
+Temporal has built-in Nexus machinery that performs the low-level Nexus RPC operations on the wire and provides an integrated Temporal SDK experience to create Nexus Services and use them from a caller Workflow via a Nexus Endpoint in the Nexus API Registry.
+
+**Nexus API Registry:** A `Nexus API Registry` is a list of Nexus `Endpoints` available within a Temporal environment. 
+Adding a Nexus `Endpoint` allows Nexus Operation requests to be sent to the Worker listening on the Endpoint’s target Namespace and Task Queue, from a caller Workflow using the name of the Endpoint in the Nexus API Registry.
+
+**Nexus Endpoints:** A Nexus `Endpoint` is like an API proxy that can serve one or more Nexus `Services` from a `Worker` listening on a target Namespace and Task Queue. 
+This enables presenting a clean API contract and hiding the underlying Temporal implementation, which may consist of internal `Workflows`, `Signals`, and other Temporal primitives – or arbitrary code. 
+In Temporal Cloud an endpoint’s access-control policy determines which callers are allowed to access a Nexus `Endpoint`.
+
+**Nexus Services:** A Nexus `Service` is a named collection of arbitrary-duration Nexus `Operations` that provide an API contract suitable for sharing across team boundaries. 
+Nexus Services are registered with a `Worker` that is listening on the target Namespace and Task Queue for an `Endpoint`.
+
+**Nexus Operations:** A Nexus `Operation` is not intrinsically durable, but can be made durable by backing it with Temporal primitives like a `Workflow`, `Update`, or `Signal`.
+The Temporal SDK provides helper functions to create Nexus Operations that may be registered with a Nexus Service in a `Worker`. 
+Synchronous Nexus `Operations` can also execute arbitrary code, but unlike `Activities` they have a short request timeout (under 10 seconds) and don’t support [heartbeats](https://community.temporal.io/t/best-practices-for-long-running-activities/934).
```

Comment:
Should we pull this community article into docs instead of linking to it? Feels odd linking to a community forum.

Reply:
Taking out the core components section for now since there's already sections dedicted for each one.